supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -546,13 +546,46 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[ip]]
 ==== image:images/yes.png[yes] IP (noun)
-*Description*: "IP" is an abbreviation for "Internet Protocol".
+*Description*: "IP" is an abbreviation for "Internet Protocol". Use "IP" if refer to the Internet Protocol in general if the specific versions, IPv4 and IPv6, do not matter. Use "IP address" instead of "IP" when writing about IP addresses. Do not expand the abbreviation on the first usage.
```

Comment:
Should "if" be "to"?
```suggestion
*Description*: "IP" is an abbreviation for "Internet Protocol". Use "IP" to refer to the Internet Protocol in general if the specific versions, IPv4 and IPv6, do not matter. Use "IP address" instead of "IP" when writing about IP addresses. Do not expand the abbreviation on the first usage.
```